Templates/Ros2RoboticManipulationTemplate/Template/project.json

```diff
@@ -0,0 +1,27 @@
+{
+    "project_name": "${Name}",
+    "version": "1.0.0",
+    "project_id": "{C930C5B9-4B70-40B7-88F4-A7E927D6A677}",
+    "origin": "The primary repo for ${Name} goes here: i.e. http://www.mydomain.com",
+    "license": "What license ${Name} uses goes here: i.e. https://opensource.org/licenses/Apache-2.0 Or https://opensource.org/licenses/MIT etc.",
+    "display_name": "${Name}",
+    "summary": "A short description of ${Name}.",
+    "canonical_tags": [
+        "Project"
+    ],
+    "user_tags": [
+        "${Name}"
+    ],
+    "icon_path": "preview.png",
+    "engine": "o3de",
+    "external_subdirectories": [
+        "Gem"
+    ],
+    "restricted": "MySimulationProject",
+    "engine_version": "2.1.0",
+    "gem_names": [
+        "ProteusRobot",
```

Comment:
Update & remove assets it does not depend on.

Reply:
Can be resolved in a follow-up PR

Reply:
Added issue : https://github.com/o3de/o3de-extras/issues/466